common/src/test/scala/uk/ac/wellcome/models/ItemTest.scala

```diff
@@ -23,13 +24,13 @@ class ItemTest extends FunSpec with Matchers with JsonTestUtil {
       |  "locations": [
       |    {
       |      "locationType": "location",
+      |      "url" : null,
+      |      "credit" : null,
```

Comment:
circe is configured to output everything, even nulls. We can configure it differently but I prefer this 'cause it means all fields are always present